elements/pfe-select/src/pfe-select.js

```diff
@@ -80,7 +80,7 @@ class PfeSelect extends PFElement {
     super.disconnectedCallback();
 
     this.observer.disconnect();
-    this._input.removeEventListener("input", this._inputChanged);
+    this._input && this._input.removeEventListener("input", this._inputChanged);
```

Comment:
This fixes a bug where if pfe-select doesn't have any content it breaks JS because there's no `this._input` to remove the event listener on.

Reply:
```suggestion
    if (this._input) this._input.removeEventListener("input", this._inputChanged);
```
What would you think of this for a slight readability improvement

Reply:
I'm not a fan of one line `if()` statements but I don't feel that strongly. If that's how it is elsewhere in the code base I'm cool changing it 😎

Reply:
I'm cool if you want to wrap it, just thinking the if is easier to understand than the amper syntax

Reply:
Done!